app/views/internal/mods/index.html.erb

```diff
@@ -1,33 +1,29 @@
-<div class="row m-3">
+<div class="row my-3">
   <div class="col">
     <ul class="nav nav-pills">
       <li class="nav-item">
         <a href="/internal/mods" class="nav-link <%= "active" if params[:state].blank? %>">General Community</a>
       </li>
       <li class="nav-item">
-        <a href="/internal/mods?state=tag" class="nav-link <%= "active" if params[:state] == "tag" %>">Tag Mods</a>
+        <a href="/internal/mods?state=tag_moderator" class="nav-link <%= "active" if params[:state] == "tag_moderator" %>">Tag Mods</a>
```

Comment:
you can switch to `internal_mods_path(state: :tag_moderator)`

Reply:
I think I could go with plain `link_to` helper on that. 😉